splinepy/helpme/visualize.py

```diff
@@ -15,7 +15,7 @@
         "knot_lw",
         "Line width of knots. Number of pixels. "
         "Applicable to para_dim > 1.",
-        (int),
+        (int,),
```

Comment:
can also be a float, not so relevant if used in conjunction with the new vedo release but before you needed lw of 0.1 etc. I am not sure if we should add it now since it does not serve a useful purpose anymore. More just a find I made. (I have tested this just with he k3d backend in vedo so I am not even sure if this holds for the vtk backend.
```suggestion
        (int,float),
```